frontend/app/mocks/subscription-api.server.ts

```diff
@@ -74,5 +84,35 @@ export function getSubscriptionApiMockHandlers() {
 
       return HttpResponse.text(null, { status: 204 });
     }),
+    //MOCK TO VALIDATE THE CONFIRMATION CODE VS THE ONE ENTERED BY THE USER
+    http.post('https://api.example.com/v1/users/codes/verify', async ({ params, request }) => {
```

Comment:
Two comments:

- **Nitpick:** no need for the comment (especially in all caps). The API endpoint is self-documenting enough in my opinion.
- The current naming scheme for the `users` resource uses the `userId` in the path as the unique identifier (ie. `/users/{userId}/subscriptions`). Having an endpoint like `/users/codes` suggests that there is a user named `codes`. It might be better to remove the `/users` section in the path for consistency with the existing scheme.